packages/discord.js/src/index.js

```diff
@@ -26,7 +26,7 @@ exports.ChannelFlagsBitField = require('./util/ChannelFlagsBitField');
 exports.Collection = require('@discordjs/collection').Collection;
 exports.Constants = require('./util/Constants');
 exports.Colors = require('./util/Colors');
-exports.DataResolver = require('./util/DataResolver');
+__exportStar(require('./util/DataResolver.js'), exports);
```

Comment:
Isn't this a breaking change? Also, is the `.js` intended? 👀

Reply:
This class is private. We can do what we like. We can even remove the export.

Reply:
On that note, why is it even being exported? It's been `@private` since implementation (#1892). Shall we just remove it?

Reply:
Well, it's not like we can ask the author or the reviewers anymore about it... 😅

Reply:
> Isn't this a breaking change (not exporting `DataResolver`)?

Yep, it is